### DISCUSSION TITLE
How to place markers on crazyflie: Vicon

### DISCUSSION THREAD

**Discussion by francesco2706** [bitcraze/discussions#1617] [Q&A]
Hello bitcraze!

I'm working on a project involving drone position tracking using Vicon. 
I want to place 4 markers on the crazyflie (I have 9.5 mm Sphere ones available with a 17 mm plastic base). Since placing them on the drone they touch the propeller, I wonder if anyone has a CAD available that can be printed with a 3D printer.

**Comment by gemenerik** [comment]
Hi Francesco! Are you currently attaching the markers directly to the drone? The[ motion capture marker deck](https://www.bitcraze.io/products/motion-capture-marker-deck/) is designed for Qualisys markers, so I’m not certain about compatibility, especially since yours sound fairly large. If you can mount the markers on the deck and that still doesn't fit, you might indeed need a  custom solution to extend them beyond the propellers, like you suggested, but that could be problematic for flight.

**Reply by gemenerik** [reply]
If you can spare the budget, it might be a better option to just get some smaller markers.
